docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -50,6 +50,10 @@ Tooltip options control the information overlay that appears when you hover over
 
 {{< docs/shared lookup="visualizations/tooltip-mode.md" source="grafana" version="<GRAFANA VERSION>" >}}
 
+### Hover proximity
+
+This option controls how close your cursor must be to a data point before the tooltip appears.
```

Comment:
I feel like we should be explicit the measurement about this as I've definitely wondered about this at times. Feel free to change my wording.
```suggestion
This option controls how close your cursor must be to a data point before the tooltip appears. Values are in pixels.
```